cloudformation.yaml

```diff
@@ -172,6 +172,16 @@ Resources:
             touch /var/log/identity-gateway.log
             chown identity-gateway:identity-gateway /var/log/identity-gateway.log
 
+            # Try multiple times to get parameter store values. The SSM command can fail when called immediately on instance startup.
+            while true; do
```

Comment:
Maybe a sleep and a kill switch?

Reply:
I've added a sleep.

If parameter store isn't available it's probably okay for this to loop forever, writing to stderr until the instance is deemed unhealthy